README.md

```diff
@@ -22,4 +22,4 @@ Alert boxes are achieved by using a block quote that has one of 'warn', 'danger'
 ## Join the Unofficial Discord API Server
 The Unofficial Discord API server is a common hangout for library and bot developers alike. It's a great starting point for those looking to dive in and learn bot-creation with the Discord API.
 
-[![](https://discord.com/api/guilds/81384788765712384/embed.png?style=banner1)](https://discord.gg/discord-api)
+[![](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner1)](https://discord.gg/discord-api)
```

Comment:
```suggestion
[![Widget for the Discord API guild](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner1)](https://discord.gg/discord-api)
```
Might just be worth adding alt text in case anyone is partially sighted, given the announcement that Discord is trying to be more inclusive of marginalised groups from a few months ago

Reply:
Thank you, great callout. Realistically all the images in the docs should have this too.

Reply:
@msciotti, I have put a PR up targeting this branch that completes adding this alt text across this documentation, if you decide to pull that into this:

https://github.com/discord/discord-api-docs/pull/2106.